lib/noun-project-api/icon.rb

```diff
@@ -1,35 +1,36 @@
 module NounProjectApi
+  # A single Icon as an abstracted ruby object.
   class Icon
     PREVIEW_SIZE_200 = 200
     PREVIEW_SIZE_42 = 42
     PREVIEW_SIZE_84 = 84
 
-    PUBLIC_DOMAIN_LICENSE = "public-domain"
+    PUBLIC_DOMAIN_LICENSE = 'public-domain'
 
     attr_accessor :original_hash
 
     def initialize(origin)
       origin = JSON.parse(origin) if origin.is_a? String
-      origin = origin.delete("icon") if origin.key? "icon"
+      origin = origin.delete('icon') if origin.key? 'icon'
 
       @original_hash = origin
     end
 
     def id
-      original_hash["id"].to_i
+      original_hash['id'].to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.